lib/src/over_react_test/common_component_util.dart

```diff
@@ -112,6 +131,9 @@ void commonComponentTests(BuilderOnlyUiFactory factory, {
   }
 }
 
+Iterable _flatten(Iterable iterable) =>
+    iterable.expand((item) => item is Iterable ? _flatten(item) : [item]);
+
```

Comment:
This appears to be unused

Reply:
Good catch! Added here: https://github.com/Workiva/over_react_test/pull/95/files?w=1#diff-6dde553a16b2739f0b38df1918ceb060R104 (Moved from https://github.com/Workiva/over_react_test/pull/95/files?w=1#diff-6dde553a16b2739f0b38df1918ceb060L92)